src/store.js

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable import/prefer-default-export */
+
+import { configureStore } from '@reduxjs/toolkit';
+
+export const store = configureStore({
+  reducer: {},
+});
```

Comment:
- [ ] [OPTIONAL] Please remove this file as you already have a `store` file for the redux.